website/documentation/getting-started/common-pitfalls/_index.md

```diff
@@ -0,0 +1,209 @@
+- specify how the checks are executed (service)
+- webhook serving pod: perform a review and reply to the `admissionReview` request
+
+![](./images/user-webhook-fail.gif)
+
+What can go wrong (i.e., the webhook is not reachable and thus blocking the creation of the pod or any other operation that the webhook is responsible for)?
+
+- VPN / connection issues can happen both on the side of the seed as well as the shoot
+- The traffic cannot reach the pod (network issue, pod not available)
+- The pod is processing too slow (e.g., because there are too many requests)
+
+### Timeout
+
+![](./images/timeout.png)
+
+Webhooks are a very helpful feature of Kubernetes. However, they can easily be configured to break a shoot cluster. Take the timeout, for example. High timeouts (>15s) can lead to blocking requests of control plane components. That's because most control-plane API calls are made with a client-side timeout of 30s, so if a webhook has `timeoutSeconds=30`, the overall request might still fail as there is overhead in communication with the API server and other potential webhooks.
+
+{{% alert color="info"  title="Recommendation" %}}
+Webhooks (esp. mutating) may be called sequentially and thus adding up their individual timeouts. Even if the `faliurePolicy` is ignored, the timeout will stop the request.
+{{% /alert %}}
+
+### Recommendations
+
+![](./images/webhook-recommendations.png)
+
+Problematic webhooks are reported as part of a shoot's status. In addition to timeouts, it is crucial to exclude the `kube-system` namespace and (potentially non-namespaced) resources that are necessary for the cluster to function properly. Those should not be subject to a user-defined webhook. 
+
+**Example:**
+
+A webhook checks node objects upon creation and has a `failurePolicy: fail`. If the webhook does not answer in time (either due to latency or because there is no pod serving it), new nodes cannot join the cluster.
+
+For more information, see [Shoot Status](https://github.com/gardener/gardener/blob/master/docs/usage/shoot_status.md#constraints).
+
+## Conversion Webhooks
+
+### Who installs a conversion webhook?
+
+![](./images/conversion-webhook-install.png)
+
+If you have written your own CRD and made a version upgrade, you will also have consciously written & deployed the conversion webhook.
+
+However, sometimes, you simply use helm or kustomize to install a (third-party) dependency that contains CRDs as well. Of course, those can contain conversion webhooks as well. As a user of a cluster, please make sure to be aware what you deploy.
+
+### CRD with a Conversion Webhook
+
+![](./images/conversion-webhook-crd.png)
+
+Conversion webhooks are tricky. Similarly to regular webhooks, they should have a low timeout. However, they cannot be remediated automatically and can cause strange errors. For example, if a webhook is invoked but not available, it can block the garbage collection run by the kube-controller-manager. 
```

Comment:
```suggestion
Conversion webhooks are tricky. Similarly to regular webhooks, they should have a low timeout. However, they cannot be remediated automatically and can cause errors in the control plane. For example, if a webhook is invoked but not available, it can block the garbage collection run by the kube-controller-manager. 
```